### ISSUE TITLE
Default thumbnail asset not found

### ISSUE THREAD

**Issue by wpbonelli** [spatialaudio/nbsphinx#796]
I'm using nbsphinx to a create a notebook [gallery](https://modflow6-examples.readthedocs.io/en/latest/notebook_examples.html). I am using `nbsphinx_execute = "never"` and converting python scripts to notebooks with `jupytext` in GitHub Actions CI, then uploading the notebooks for ReadTheDocs to consume. The reason for this is to avoid timeouts on RTD, some of the notebooks are slow.

Note the empty thumbnails for some of the notebooks, for instance the "Advgwtidal example". I see a request going out for the thumbnail and returning 404:

```
GET https://modflow6-examples.readthedocs.io/en/latest/_images/_notebooks_ex-gwf-advtidal_10_5.png
```

Opening the notebook page, all the plots load fine via:

```
GET https://modflow6-examples.readthedocs.io/en/latest/_images/_notebooks_ex-gwf-advtidal_10_1.png
GET https://modflow6-examples.readthedocs.io/en/latest/_images/_notebooks_ex-gwf-advtidal_10_2.png
GET https://modflow6-examples.readthedocs.io/en/latest/_images/_notebooks_ex-gwf-advtidal_10_3.png
GET https://modflow6-examples.readthedocs.io/en/latest/_images/_notebooks_ex-gwf-advtidal_10_4.png
```

Why is the thumbnail index 5, when there are only 4 plots in the notebook, and nbsphinx is not re-executing the notebook?

A locally built/served site (with latest sphinx and nbsphinx) shows the correct thumbnails, and the thumbnail retrieved for the gallery page is the same asset as the one shown in the notebook (the index is 626, I presume because I've run the notebook many times).

**Comment by mgeier** [comment]
I think it works with `nbconvert==7.16.3`, but it doesn't work with `nbconvert<7.14` nor with `nbconvert==7.16.4`.

**Comment by wpbonelli** [comment]
nbconvert 7.16.3 does work! thanks for the tip.

**Comment by mgeier** [comment]
#797 should fix this problem for `nbconvert>=7.14`.

**Comment by mgeier** [comment]
> converting python scripts to notebooks with `jupytext` in GitHub Actions CI

Out of curiosity, how exactly do you execute your notebooks?

When I try executing a notebook with two consecutive stream outputs, those get merged into one stream output. So far I wasn't able to create a notebook with consecutive stream outputs (which means that this issue cannot arise in the first place).

**Comment by wpbonelli** [comment]
> how exactly do you execute your notebooks?

With jupytext e.g. `jupytext --from py --to ipynb --execute ...`, invoked from python with `Popen(...)`, driven by pytest executing in parallel via xdist. [Here](https://github.com/MODFLOW-USGS/modflow6-examples/actions/runs/9056301514/job/24878635139#step:12:1) is a CI run. It's a bit goldberg...

**Comment by wpbonelli** [comment]
> BTW, removing this output would also have solved the thumbnail problem.

I didn't notice the connection between missing thumbnails and text output in the cell that creates the plot. Thanks!

> How/where it the following line in your output created?

Deep inside the flopy package, I will have to dig for it &mdash; it should really be suppressed in any case
